pyaedt/generic/general_methods.py

```diff
@@ -1770,6 +1770,36 @@ def __init__(self):
         if is_linux:
             self._aedt_environment_variables["ANS_NODEPCHECK"] = "1"
         self._desktop_launch_timeout = 90
+        self._aedt_process_id = None
+        self._is_student = False
+
+    @property
+    def aedt_process_id(self):
+        """Set the desktop process id. Default is ``None``.
+
+        Returns
+        -------
+        int
+        """
+        return self._aedt_process_id
+
+    @aedt_process_id.setter
+    def aedt_process_id(self, value):
+        self._aedt_process_id = int(value)
+
+    @property
+    def is_student(self):
+        """Set the desktop process is student version. Default is ``False``.
```

Comment:
```suggestion
        """Whether the desktop process is set to the student version. The
        default is ``False``.
```
Do we need to specify what the default is here since it is not where you set the value but rather you are returning the value that is already set?